djstripe/models.py

```diff
@@ -125,7 +126,9 @@ class Customer(StripeCustomer):
 
     # account = models.ForeignKey(Account, related_name="customers")
 
-    default_source = models.ForeignKey(StripeSource, null=True, related_name="customers")
+    default_source = models.ForeignKey(
```

Comment:
Yup!

`djstripe.tests.test_event_handlers.TestCustomerEvents.test_customer_default_source_deleted` checks to see that customer does in fact still exist (but no longer has a default source).